warehouse/templates/manage/account.html

```diff
@@ -484,25 +484,113 @@ <h2>API tokens <span class="badge badge--warning">Beta feature</span></h2>
   <hr>
 
   <section id="account-events">
-    <h2>Recent account events</h2>
+    <h2>Security history <span class="badge badge--warning">Beta feature</span></h2>
 
     {% set recent_events = user.recent_events %}
     {% if recent_events|length > 0 %}
-    <table>
+
+    {% macro event_summary(event) -%}
```

Comment:
NB: I think we could probably come up with a Jinja2 filter here to perform event prettification for us; I can take a look at that in a bit.

Reply:
that would be great :grin: